feedback-front/static/js/feedback.js

```diff
@@ -41,14 +41,36 @@ var FeedBackClient = function(name) {
           headers: {
             "Content-Type" : "application/json"
           },
-          success: function(data) {
+          success: function(jqxhr) {
             // Display success message in UI
-            console.log(data);
-            console.log("Success!");
+            $('#labelNameStatus').html("<div style='color:green'>Label Name was added successfully.</div>");
+            window.setTimeout(function(){location.reload()}, 1500);
           }
-        }).fail(function (jqxhr) {
+        }).fail(function (jqxhr, exception) {
             // Display failure message in UI
-            console.log(" Adding new label value failed!");
+                console.log(" Adding new label value failed!");
+              if (jqXHR.status === 0) {
+                  $('#labelNameStatus').html("<div style='color:red'>Not connecting, verify your network</div>");
```

Comment:
What are you doing here?
do this: define a `var msg` and set it to whatever is the content of the div - for ex `Not connecting, ...` etc. 
At the end of the if-else-if ladder do a ` $('#labelNameStatus').html("<div style='color:red'>" + msg +"</div>");`